ci/build_docs.sh

```diff
@@ -25,7 +25,7 @@ rapids-mamba-retry install \
   --channel "${PYTHON_CHANNEL}" \
   rmm librmm
 
-export RAPIDS_VERSION_NUMBER="24.04"
+export RAPIDS_VERSION_NUMBER="$(sed -E -e 's/^([0-9]{2})\.([0-9]{2})\.([0-9]{2})/\1.\2/' VERSION)"
```

Comment:
Since this little sed script is used more than once, could it be put somewhere as a utility and documented?

Reply:
Opened https://github.com/rapidsai/gha-tools/pull/97.

Reply:
Awesome. I'll approve assuming you will merge this after adding calls to that script.